packages/x-components/src/x-modules/next-queries/components/next-query.vue

```diff
@@ -4,6 +4,7 @@
     :suggestion="suggestion"
     :suggestionSelectedEvents="events"
     data-test="next-query"
+    feature="next_query"
```

Comment:
I'm missing the validation of this feature in the spec file, as you did with the rest.